package-lock.json

```diff
@@ -5261,15 +5261,6 @@
           "dev": true,
           "optional": true
         },
-        "string_decoder": {
```

Comment:
The `package-lock.json` shouldn't be part of this PR

Reply:
Done